packages/api/package.json

```diff
@@ -14,7 +14,7 @@
     "test-coverage": "nyc npm test",
     "build": "webpack",
     "watch": "webpack --progress -w",
-    "serve": "LOCALSTACK_HOST=localhost NODE_ENV=test FAKE_AUTH=true cumulus-api serve",
+    "serve": "LOCALSTACK_HOST=localhost LOCAL_ES_HOST=localhost NODE_ENV=test FAKE_AUTH=true cumulus-api serve",
```

Comment:
What problem does this solve and why didn’t we need to solve it before? Just curious.

Reply:
I ... think @markdboyd  spent some time tracking down that this needed set to run the local API.    In anycase,  from the docs, the only instance of running npm-run-serve is :

### Running the API locally

To run the API locally using Localstack for AWS services without Earthdata authentication required:

```bash
  $ npm run serve
```

As such you *must* have LOCAL_ES_HOST set to the IP of your test stack, else running it will fail (no elasticsearch).   I'm presuming 'localhost' is a safe/sane default for development environments, given the way the unit test stack is setup, and avoids minor annoyance in setting that env variable which doing api work.    If we wanted to be defensive about it we could respect existing shell values or the like.   

It's possibly worth arguing that we should add it to `npm run serve-oauth` as well 🤔

Reply:
I actually realized that I needed it to run the unit tests locally.

I don't think you *need* to set `LOCAL_ES_HOST` to interact with API endpoints when you do `npm run serve` because the code ultimately falls back to ES server from localstack:

https://github.com/nasa/cumulus/blob/master/packages/api/es/search.js#L51

But it's better to make this explicit instead of relying on the way the code is currently written. And yes, we should add the same for `serve-oauth`.

Reply:
L51 is from *prod* config, this is running in test because `NODE_ENV=test`.    So it winds up picking the wrong port when it goes through

const getLocalEsHost = () => {
  if (process.env.LOCAL_ES_HOST) return `${process.env.LOCAL_ES_HOST}:9200`;
  if (process.env.LOCALSTACK_HOST) return `${process.env.LOCALSTACK_HOST}:4571`;
  return 'localhost:9200';
};

const esTestConfig = () => ({
  host: getLocalEsHost(),
  requestTimeout: 5000
});

Reply:
Added 77f6f8b7f